jerry-core/jerry.c

```diff
@@ -125,9 +125,18 @@ jerry_make_api_unavailable (void)
 } /* jerry_make_api_unavailable */
 
 /**
+ * Returns whether the given jerry_value_t is error.
+ */
+inline bool __attr_always_inline___
+jerry_value_is_error (const jerry_value_t value) /**< api value */
+{
+  return ECMA_IS_VALUE_ERROR (value);
+} /* jerry_value_is_error */
+
+/**
  * Returns whether the given jerry_value_t is null.
  */
-bool
+inline bool __attr_always_inline___
```

Comment:
I think this depends on LTO/all-in-one build, and an API user unlikely to have this advantage.